### ISSUE TITLE
Suggestion: load dashHtmlComponents package in R example

### ISSUE THREAD

**Issue by rpkyle** [facultyai/dash-bootstrap-components#370]
In the current R documentation available [here](https://dash-bootstrap-components.opensource.faculty.ai/docs/dashr/), the simple example does not reference `dashHtmlComponents`, which it seems to require be loaded before `dbcContainer` is invoked. The suggested codeblock is

```
library(dash)
library(dashHtmlComponents)
library(dashBootstrapComponents)

app <- Dash$new(external_stylesheets = dbcThemes$BOOTSTRAP)

app$layout(dbcContainer(dbcAlert("Hello Bootstrap!", color = "success"),
                        className = "p-5"))

app$run_server(showcase = TRUE)
```

This will avoid the following console error in a fresh R session and render the page properly:

<img width="644" alt="Screenshot 2020-05-09 17 20 25" src="https://user-images.githubusercontent.com/9809798/81485215-6dab4500-9219-11ea-8e37-aec3730f45e3.png">

**Comment by tcbegley** [comment]
Thanks @rpkyle, turns out I even had the extra line in the app I was testing locally but didn't put it in the docs somehow 😖 I've fixed the example as part of #371 

Any idea why this is happening? FWIW I tried making an app using only _dashCoreComponents_ based on an example in the readme and noticed the same error. With _dashHtmlComponents_ loaded it runs fine, without it throws an error despite no explicit dependence.

```r
library(dash)
library(dashCoreComponents)

app <- Dash$new()

app$layout(
  dccInput(id = "graphTitle", 
           value = "Let's Dance!", 
           type = "text"),
  dccGraph(id = "giraffe",
           figure = list(
             data = list(x = c(1,2,3), y = c(3,2,8), type = 'bar'),
             layout = list(title = "Let's Dance!")
           )
  )
)

app$callback(output = list(id = "giraffe", property = "figure"), 
             params = list(input("graphTitle", "value")),     
             function(newTitle) {
               
               rand1 <- sample(1:10, 1)
               
               rand2 <- sample(1:10, 1)
               rand3 <- sample(1:10, 1)
               rand4 <- sample(1:10, 1)
               
               x <- c(1,2,3)
               y <- c(3,6,rand1)
               y2 <- c(rand2,rand3,rand4)
               
               df = data.frame(x, y, y2)
               
               list(
                 data = 
                   list(            
                     list(
                       x = df$x, 
                       y = df$y, 
                       type = 'bar'
                     ),
                     list(
                       x = df$x, 
                       y = df$y2, 
                       type = 'scatter',
                       mode = 'lines+markers',
                       line = list(width = 4)
                     )                
                   ),
                 layout = list(title = newTitle)
               )
             }
)

app$run_server(showcase = TRUE)
```

**Comment by rpkyle** [comment]
> Thanks @rpkyle, turns out I even had the extra line in the app I was testing locally but didn't put it in the docs somehow 😖 I've fixed the example as part of #371
> 
> Any idea why this is happening? FWIW I tried making an app using only _dashCoreComponents_ based on an example in the readme and noticed the same error. With _dashHtmlComponents_ loaded it runs fine, without it throws an error despite no explicit dependence.

@tcbegley I think this might actually be due to outdated code that I'm working to remove within Dash for R. There's a PR open now I'm planning to wrap up next week that should remove the implicit dependency, since I'm about to send the package off to CRAN for review, and I don't want this to impact anyone else. 

With any luck, it will be resolved soon, then I'll retest with `dashBootstrapComponents` and update you here.

**Comment by rpkyle** [comment]
@tcbegley Some good news -- I've merged plotly/dashR#121, and released v0.5.0 of Dash for R. Dash no longer has any implicit dependence on `dashHtmlComponents`, so it's now possible to use the example as originally written 😅 .

I've tested and confirmed this locally, and the new version is available from `master` if you'd like to try it. If it works on your end also, feel free to close this issue!

**Comment by tcbegley** [comment]
Thanks @rpkyle! Can confirm it's all working great my end, thanks for your work on this and letting me know :)
